addons/medical_gui/stringtable.xml

```diff
@@ -3,12 +3,15 @@
     <Package name="Medical GUI">
         <Key ID="STR_ACE_Medical_GUI_Category">
             <English>ACE Medical GUI</English>
+            <German>ACE Sanitäter Oberfläche</German>
         </Key>
         <Key ID="STR_ACE_Medical_GUI_EnableActions_DisplayName">
             <English>Enable Medical Actions</English>
+            <German>Aktiviere Sanitäter Aktionen</German>
         </Key>
         <Key ID="STR_ACE_Medical_GUI_EnableActions_Description">
             <English>Enables medical actions for the Interaction Menu and selects their style.</English>
+            <German>Aktiviert die Sanitäteroptionen für das Interaktionsmenü und legt das Aussehen fest</German>
```

Comment:
Displayname says "aktionen", this says "optionen" also mix between combining with Sanitäter and not combining.
Also "Sanitäts" to not make female medics angry.